src/frontend/komponenter/Felleskomponenter/Toast/typer.ts

```diff
@@ -11,3 +9,10 @@ export enum ToastTyper {
     FANT_IKKE_FAGSAK = 'FANT_IKKE_FAGSAK',
     MANGLER_TILGANG = 'MANGLER_TILGANG',
 }
+
+export enum AlertType {
+    ERROR = 'error',
+    WARNING = 'warning',
+    INFO = 'info',
+    SUCCESS = 'success',
+}
```

Comment:
Optimaltsett burde jeg ikke trenge å lage en enum for dette, men fant ikke en tilsvarende greie i ds-biblioteket 😞

Reply:
Ser at @kristeine har brukt strengen direkte uten å lage et enum på lignende ting. 
Typ her: https://github.com/navikt/familie-ba-sak-frontend/pull/2188/files
![image](https://user-images.githubusercontent.com/17828446/179736803-474def28-c7f7-4e60-a17a-51629cd23979.png)


Jeg foretrekker egentlig at vi har et enum, så tenker dette er fint 😊